### ISSUE TITLE
Build Folder

### ISSUE THREAD

**Issue by 8brandon** [badlydrawnrob/anki#55]
I have setup the project as you mentioned and am building the html files, but actually don't know what to do with these built files.
If you can let me know what you do with these built HTML files to turn them into cards that would be great!

**Comment by badlydrawnrob** [comment]
@brandoncroberts Better late than never.

For now, please [reference this document](https://github.com/badlydrawnrob/anki/blob/master/source/docs/advanced/index.md) and the [css styles](https://github.com/badlydrawnrob/anki/blob/master/source/docs/highlight/index.md#basic-syntax-highlighting) you can format yourself ...

Right now it uses [moustache](https://www.npmjs.com/package/mustache), template files and a [`.json` file](https://github.com/badlydrawnrob/anki/blob/master/source/themes/cards/simple/simple.json) which compiles with a [command line prompt](https://github.com/badlydrawnrob/anki/blob/master/package.json) referencing [these template files](https://github.com/badlydrawnrob/anki/blob/8a1b6f75bad54131a58485538a0b217966238d75/package.json#L9).

### It's probably best not to bother right now ...

**BUT. There's a big but. I think I'm going to have to rip out the moustache functionality** as Anki has seemed to deviate away from the [mustache](https://mustache.github.io)/[pystache](https://pypi.org/project/pystache/) and [my current live preview version](https://github.com/badlydrawnrob/anki/issues/5) might not be worth the effort to keep around.

It's the biggest pain that this [`{{{ triple brackets }}}`](https://github.com/badlydrawnrob/anki/issues/27) method doesn't seem to play nice anymore with Anki, as it allows me to reuse the Anki template files and escape the json [for a nice demo display](https://github.com/badlydrawnrob/anki/blob/master/source/themes/cards/simple/simple.mustache). That approach looks like it out the window now, so I either have to:

1. Duplicate the code and keep mustache as is, or ...
2. Just use plain HTML (and likely have to duplicate the code with this method too).

I'm currently leaning towards **option 2** because it just takes away the headache of something not working how I imagined it (the json includes characters that won't render properly without `{{{ ... }}}` escaping. _[This thread seems to show an alternative way](https://stackoverflow.com/questions/39409853/how-to-change-the-dont-escape-html-delimiter-in-mustache-js)_ of doing it, but I doubt it'll work in Anki.

Time and time again, I find that **[Dependancy Hell](https://github.com/badlydrawnrob/anki/blob/master/source/docs/error/index.md)** is something that keeps me out of the programming profession. It's one thing after the other!

**TL;DR:** hold fire until I've had the chance to update the code, or keep this version and do as you want.

**Comment by badlydrawnrob** [comment]
@brandoncroberts There's also some useful information here for future reference: **[Build script and security](https://github.com/badlydrawnrob/print-first-css/issues/25)**

**Comment by badlydrawnrob** [comment]
@brandoncroberts Hopefully [this page](https://github.com/badlydrawnrob/anki/blob/master/source/docs/highlight/index.md#customising-themes) and this [advanced documentation](https://github.com/badlydrawnrob/anki/blob/master/source/docs/advanced/index.md) (plus links to Github issues) will go a long way to making the build process easier to understand?

Hopefully you're still interested! :)
